specific/cmo_purchase_level_validation/models/purchase_level_validation.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models, api, _
+from openerp.exceptions import ValidationError
+
+
+class PurchaseLevelValidataion(models.Model):
+    _name = 'purchase.level.validation'
+
+    level = fields.Integer(
+        string='Level',
+        required=True,
+    )
+    limit_amount = fields.Float(
+        string='Limit Amount',
+        required=True,
+    )
+    operating_unit_id = fields.Many2one(
+        'operating.unit',
+        string='Operating Unit',
+    )
+    user_ids = fields.Many2many(
+        'res.users',
+        'res_user_rel', 'validation_id', 'user_id',
```

Comment:
Please have more meaningful table name, i..e, res_user_purchase_level_validation_rel
It is ok to be long, as soon as it is more readable.

Reply:
done krub